### DISCUSSION TITLE
For creating Medusa Marketplace

### DISCUSSION THREAD

**Discussion by Shubhamtechcopain** [medusajs/medusa#4757] [Q&A]
I have to create medusa marketplace for multiple vendor and I was following this article -https://medusajs.com/blog/extending-medusa-usecase-marketplace/ . But when I am creating multiple user, multiple store id is creating  and assign to user, but when login with the credential same store is open. and I have check there is no member variable is creating.

**Comment by AbhinavChavareTechcopain** [comment]
please use the latest Medusa app and follow the same article for Medusa marketplace. If you have a doubt, please ping abhinavbalkrishnachavare_90980 on discord. thanks

**Comment by mustafaasif1** [comment]
I am facing the same issue. Multiple stores are being created but the same store is being opened even with different accounts. I have the latest Medusa application

**Comment by lacymorrow** [comment]
Any progress on this?

Currently the guide to create a marketplace breaks down when it gets to the postman section, there is no `members` property to the newly created stores.
